config.py

```diff
@@ -132,8 +138,16 @@ class TestingConfig(BaseConfig):
     TESTING = True
     MOCK_EMAIL = True
 
-    # Use in-memory SQLite database for testing
-    SQLALCHEMY_DATABASE_URI = "sqlite://"
+    # Using a local postgre database on MacOS
+    # SQLALCHEMY_DATABASE_URI = "postgresql:///bit_schema_test"
+
+    # Using a local postgre database on other OS
+    # SQLALCHEMY_DATABASE_URI = BaseConfig.build_db_test_uri()
+
+    # uncomment when pushing to upstream for Github action to Run test
+    SQLALCHEMY_DATABASE_URI = (
+        "postgresql://postgres:postgres@localhost:5432/bit_schema_test"
```

Comment:
We are returning a hard coded string. Maybe if we return a string based using the environment variables, contributors won't have to uncomment this. Do you think that is a good idea?

Reply:
Yep, fix it now by using the existing db_test_uri for non-MacOS lines 😉